### PR TITLE
[DEV APPROVED] TP: 8729, Comment: Disable GitHub pages

### PR DESCRIPTION
**Pull request by davidtrussler**
[TP8729](https://moneyadviceservice.tpondemand.com/RestUI/Board.aspx#page=board/5624876525867731357&appConfig=eyJhY2lkIjoiREI4ODcwRjkxMDNDRTM2NTlBMzhDNTRBRTVBNUU1N0UifQ==&searchPopup=userstory/8729)

The main work for this has already been merged but there are build problems that appear to be connected to a known issue with GitHub pages. This means that there will be no documentation for the tooltip helpers on Made with Dough but the helpers should be useable.

### CODE REVIEW

**Comment by osharpe** [comment]
👍

File: build.sh [hunk 1]
```diff
@@ -6,7 +6,8 @@ export RAILS_ENV=build
 export BUNDLE_WITHOUT="development:test"
 
 npm install
-./node_modules/gulp/bin/gulp.js build
+# Temporary fix to disable github pages which is causing build problems
```
Comment (Tr4pSt3R) [inline build.sh]: Hi @davidtrussler is there a ticket to fix this later so that it doesn't end up being a permanent _temp_??

**Comment by thecountgs** [comment]
👍

File: build.sh (hunk 1 above)
Comment (davidtrussler) [reply build.sh]: Good question. If it works there should be.

**Pull request merged**